### ISSUE TITLE
NuGet: need to update AfterLutz package metadata

### ISSUE THREAD

**Issue by Aaronontheweb** [AfterLutz/Akkatecture#64]
There are a few areas of NuGet metadata that need to be updated so users of the `.AL` NuGet packages get the right information about the current status / home / maintainers of the project:

https://github.com/AfterLutz/Akkatecture/blob/b7c7fd002b2af4002572b029ba58b1356dc7047b/Directory.Build.props#L3-L5 

https://github.com/AfterLutz/Akkatecture/blob/b7c7fd002b2af4002572b029ba58b1356dc7047b/Directory.Build.props#L8

**Comment by SeanFarrow** [comment]
Happy to do this, if it would help. One question, should we dynamically generate the end copyright year, so that we don't have to keep updating it when a new year roles around?

**Comment by malclear** [comment]
I think automating the adjustment of year would be great. 
A question I have is regarding how we should maintain Lutando's copyright information. I know it's common to list previous authors in the individual code pages, but should we update his copyright dates there as well every year?
@Aaronontheweb, thoughts?

**Comment by Aaronontheweb** [comment]
I would add a copyright header that preserve's Lutando's copyright but adds a new one for "AfterLutz Contributors" or the "AfterLutz Project" - that's probably the best way to do it.

**Comment by SeanFarrow** [comment]
Yep, that’s what makes complete sense to me.

**Comment by malclear** [comment]
I agree we should keep his name there. Should we increment the year on
Lutando's copyright each year, or freeze it at 2021?

*Malcolm Learner*
***@***.***
(214) 208-7005

**Comment by Aaronontheweb** [comment]
Freeze it

**Comment by SeanFarrow** [comment]
Yep, definitely agree we should freeze it.
Can you have two copyright tags in a nuspec file? I’ve never seen that done, so am thinking the answer is no.

**Comment by Aaronontheweb** [comment]
> Can you have two copyright tags in a nuspec file? I’ve never seen that done, so am thinking the answer is no.

No. I'd just delimit them via commas.
